experimental/packages/otlp-transformer/package.json

```diff
@@ -53,7 +53,8 @@
   ],
   "peerDependencies": {
     "@opentelemetry/api": ">=1.3.0 <1.5.0",
-    "@opentelemetry/api-logs": "0.38.0"
+    "@opentelemetry/api-logs": "0.38.0",
+    "@opentelemetry/sdk-logs": "0.38.0"
```

Comment:
I think this should be in `dependencies`? If a user is using other signals only they may not have the sdk-logs package installed